WebApplication/Services/ProjectService.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using WebApplication.Definitions;
+using WebApplication.Services.Exceptions;
+using WebApplication.State;
+using WebApplication.Utilities;
+
+namespace WebApplication.Services
+{
+    public class ProjectService
+    {
+        private readonly ILogger<ProjectService> _logger;
+        private readonly UserResolver _userResolver;
+        private readonly Uploads _uploads;
+
+        public ProjectService(ILogger<ProjectService> logger, UserResolver userResolver, Uploads uploads)
+        {
+            _logger = logger;
+            _userResolver = userResolver;
+            _uploads = uploads;
+        }
+
+        public async Task<string> CreateProject(NewProjectModel projectModel)
```

Comment:
Since you extracting business logic - it would be good to not use `NewProjectModel` here. BLL should not know about `IFormFile`, right?

Reply:
Yes it is correct. My plan was to refactor a bit more. But first I would like to prepare working version of the PoC.